src/benchmark_api_internal.cc

```diff
@@ -79,7 +83,7 @@ BenchmarkInstance::BenchmarkInstance(Benchmark* benchmark,
   }
 }
 
-double BenchmarkInstance::MinTime() const {
+double BenchmarkInstance::min_time() const {
```

Comment:
nit: this should be `MinTime` per google style.

Reply:
Done